sdk/core/core-rest-pipeline/core-rest-pipeline.shims.d.ts

```diff
@@ -0,0 +1,8 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
Do we need to provide an additional shim for the downleveled rolledup declarations file?

Reply:
Yep, though I was hoping we wouldn't.